cmd/tusd/cli/log.go

```diff
@@ -16,22 +16,37 @@ func SetupStructuredLogger() {
 		level = slog.LevelDebug
 	}
 
-	slogger := slog.New(
-		slog.NewTextHandler(logWriter{stdout}, &slog.HandlerOptions{
+	replaceAttrFunc := func(groups []string, a slog.Attr) slog.Attr {
+		// Remove time attribute, because that is handled by the logger
+		if a.Key == slog.TimeKey {
+			return slog.Attr{}
+		}
+		// Rename `msg` to `event`
+		if a.Key == slog.MessageKey {
+			a.Key = "event"
+		}
+		return a
+	}
+
+	var handler slog.Handler
+	switch Flags.OutputFormat {
+	case "json":
+		handler = slog.NewJSONHandler(logWriter{log.New(os.Stdout, "", 0)}, &slog.HandlerOptions{
```

Comment:
```suggestion
		handler = slog.NewJSONHandler(os.Stdout, &slog.HandlerOptions{
```

I don't think we need `logWriter` here. That is just used for the text format to get the desired behavior we want. Or does that not work then?

Reply:
the argument needs to be an io.Writer instance.

Reply:
`os.Stdout` is of type `*os.File` which implements `io.Writer`, doesn't it?

Reply:
Correct, fixed